docs/source/galleries/tutorials/04_spike_sorting.py

```diff
@@ -197,9 +198,11 @@
 sorting_dict = {"sorting": {"mountainsort5": {"filter": False}}}
 
 session.sort(
-    configs=sorting_dict, run_sorter_method="local"
+    configs={"mountainsort5": {"filter": False}},  
```

Comment:
I think it's okay as it was as `sorting_dict` is defined on line 198. It could go either way, but is slightly more readable to define it first as a variable.